performance/performance_tuning.rst

```diff
@@ -241,14 +239,14 @@ There are two configuration parameters which relate to the format in which inter
 
 However, for certain data types like hll or hstore arrays, the cost of serializing and deserializing data is pretty high. In such cases, using binary format for transferring intermediate data can improve query performance due to reduced CPU usage. There are two configuration parameters which can be used to tune this behaviour, citus.binary_master_copy_format and citus.binary_worker_copy_format. Enabling the former uses binary format to transfer intermediate query results from the workers to the coordinator while the latter is useful in queries which require dynamic shuffling of intermediate data between workers.
 
-Real Time Executor
+Adaptive Executor
```

Comment:
Just a question: is the real-time executor entirely removed now, or merely deprecated?

Reply:
afaik removed

Reply:
but configuration is still there for backward compatibility